R/regression.R

```diff
@@ -660,20 +660,27 @@ Regression <- function(formula = as.formula(NULL),
     if (output %in% c("Jaccard Coefficient", "Correlation"))
     {
         labels <- rownames(result$summary$coefficients)[-1]
+        extracted <- ExtractCommonPrefix(labels)
+        if (!is.na(extracted$common.prefix))
+            labels <- extracted$shortened.labels
         if (partial) # missing = "Use partial data (pairwise correlations)"
         {
             .estimation.data <- data
             .weights <- weights
         }
         if (output == "Jaccard Coefficient")
         {
-            result$importance <- computeJaccardCoefficientOutput(input.formula, .estimation.data, .weights, labels)
+            result$importance <- computeJaccardCoefficientOutput(input.formula, .estimation.data,
+                                                                 .weights, labels, correction)
             result$importance.type <- "Jaccard Coefficient"
+            result$relative.importance <- result$importance
```

Comment:
Need to update `@return` in the documentation. can say it's for backwards compatibility if that's the case